platforms/software/testing-base/src/main/java/org/gradle/api/internal/tasks/testing/DefaultJUnitXmlReport.java

```diff
@@ -33,14 +31,7 @@ public DefaultJUnitXmlReport(String name, Describable owner) {
     }
 
     @Override
-    public boolean isOutputPerTestCase() {
-        return outputPerTestCase;
-    }
-
-    @Override
-    public void setOutputPerTestCase(boolean outputPerTestCase) {
-        this.outputPerTestCase = outputPerTestCase;
-    }
+    public abstract Property<Boolean> getOutputPerTestCase();
```

Comment:
🤔 Since original method was a primitive `boolean`, let's also set `convention` in constructor, like:
```
this.getOutputPerTestCase().convention(false);
```